xtask/src/commands/test.rs

```diff
@@ -6,8 +6,11 @@ use xtask::*;
 
 const TEST_DEFAULT_ARGS: &[&str] = &[
     "test",
-    "--workspace",
     "--locked",
+    "-p",
+    "apollo-router",
+    "-p",
+    "apollo-router-core",
```

Comment:
```suggestion
```

If you remove --workspace it will use the default-members.

Can you also remove xtask from the default members?

Reply:
xtask is not in the default-members so it should be all good https://github.com/apollographql/router/blob/main/Cargo.toml#L2

Reply:
cool! so you can omit the -p with the crates then :ok_hand:

Reply:
sure